tests/ci/cdk/cdk/codebuild/github_ci_linux_arm_omnibus.yaml

```diff
@@ -53,3 +53,11 @@ batch:
         privileged-mode: false
         compute-type: BUILD_GENERAL1_LARGE
         image: AWS_ACCOUNT_ID_PLACEHOLDER.dkr.ecr.AWS_REGION_PLACEHOLDER.amazonaws.com/ECR_REPO_AARCH_PLACEHOLDER:amazonlinux-2_gcc-7x_latest
+
+    - identifier: amazonlinux2_gcc7x_aarch_valgrind
+      buildspec: ./tests/ci/codebuild/linux-aarch/amazonlinux-2_gcc-7x_valgrind.yml
+      env:
+        type: ARM_CONTAINER
+        privileged-mode: false
```

Comment:
There a reason why this one isn't running in privileged mode, while the x86 one is?

Reply:
I don't think we need this to run in privileged mode. So I just picked the least permissive.

Reply:
But it is needed to run in privileged mode for x86?

Reply:
The other targets run in privileged mode, so I assume it is needed for the tests to run successfully. I copied that config for Valgrind, since it also runs the tests.